test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -36,6 +36,11 @@
 PT_EC2_SINGLE_GPU_INSTANCE_TYPE = get_ec2_instance_type(
     default="p3.2xlarge", processor="gpu", filter_function=ec2_utils.filter_only_single_gpu,
 )
+PT_EC2_GPU_INDUCTOR_INSTANCE_TYPES = []
+for instance_type in ["p3.2xlarge", "p4d.24xlarge", "g5.4xlarge", "g4dn.xlarge"]:
```

Comment:
Please move the inductor test to a separate file with inductor in the filename. upsize g4dn to 4xl.

Reply:
fixed.